test/system/271-tcp-cors-server.bats

```diff
@@ -16,13 +16,18 @@ SOCKET_FILE="$UNIT_DIR/$SERVICE_NAME.socket"
 @test "podman system service - tcp CORS" {
     skip_if_remote "system service tests are meaningless over remote"
     PORT=$(random_free_port 63000-64999)
-    $PODMAN system service --cors="*" tcp:$SERVICE_TCP_HOST:$PORT -t 20 &
+    og=${PODMAN_TMPDIR}/system-service.log
```

Comment:
Missing 'l' in 'log'

Reply:
Fixed that, but tests were still failing locally. Then I realized I never did a `make` so I had old binaries. Oops.